deepspeed-mii/README.md

```diff
@@ -0,0 +1,76 @@
+# Llama-2-chat 7B DeepSpeed MII Truss
```

Comment:
I see this example as general deepspeed-mii example, rather than llama one. In fact to serve mistral with it only thing you need to change is to provide different `repo_id` in your truss file